sci-physics/delphes/delphes-3.5.0.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake
+
+DESCRIPTION="Delphes performs a fast multipurpose detector response simulation."
+HOMEPAGE="
+	https://github.com/delphes/delphes
+	http://cp3.irmp.ucl.ac.be/projects/delphes
+"
+
+MY_P=Delphes-${PV}
```

```suggestion
MY_P=${PN^}-${PV}
```

Could write it like this to make it more fancy.